### PR TITLE
RHSTOR-3827: Fix pool compression card

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: Gowtham Shanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
image
image
image

**Comment by GowthamShanmugam** [comment]
Enabled but no PVC claim
image
image

**Comment by aruniiird** [comment]
Looks good to me...
@bipuladh , @SanjalKatiyar , please take a look...

File: packages/ocs/dashboards/block-pool/compression-details-card.tsx [hunk 1]
```diff
@@ -41,49 +42,63 @@ export const CompressionDetailsCard: React.FC = () => {
...
+    stats: Number(compressionRatio),
     isLoading: ratioLoading,
-    error: ratioError || !poolCompressionRatio,
+    error: ratioError || !capacityRatio,
     title: t('Compression ratio'),
-    getStats: () => `${humanizeNumber(poolCompressionRatio).string}:1`,
+    getStats: () =>
+      t('{{capacityRatio, number}}:1', {
+        capacityRatio: Math.round(capacityRatio),
+      }),
```
Comment (SanjalKatiyar) [inline packages/ocs/dashboards/block-pool/compression-details-card.tsx:101]: no need for translation... we can remove same from storage-efficiency-card.tsx file as well...

Comment (GowthamShanmugam) [reply packages/ocs/dashboards/block-pool/compression-details-card.tsx:101]: ack

**Review by bipuladh — APPROVED** [review]

File: packages/ocs/dashboards/block-pool/compression-details-card.tsx (hunk 1 above)
Comment (SanjalKatiyar) [reply packages/ocs/dashboards/block-pool/compression-details-card.tsx:101]: we can remove /hold label once this change is done...

**Pull request merged**